README.md

```diff
@@ -1,6 +1,9 @@
-To run:
+# [payjoin.org](https://payjoin.org)
 
-```
+Official website for the Payjoin protocol.
+
+To run locally:
+```shell
 yarn
-yarn run dev
-```
+yarn start
```

Comment:
I suppose one must run `yarn install` first

Reply:
`yarn` is a shorthand for `yarn install` and has the same effect. I changed it to `yarn install` to clear up confusion.

Reply:
TIL!